src/javascript/app_2/pages/trading/actions/helpers/contract_type.js

```diff
@@ -242,7 +242,7 @@ const ContractType = (() => {
         const start_dates_list = [];
 
         if (config.has_spot) {
-            start_dates_list.push({ text: localize('Now'), value: 'now' });
+            start_dates_list.push({ text: localize('Now'), value: 999 });
```

Comment:
Please give me any reasonable or efficient number which can represent `Now` **if you have any**. 
- We have to keep the prop type of 'start_date' as the type of `Number` only.